src/index.js

```diff
@@ -1172,13 +1248,15 @@ Firestore.log = function() {};
 Firestore.setLogFunction = function(logger) {
   validate.isFunction('logger', logger);
 
-  Firestore.log = function(methodName, varargs) {
-    varargs = Array.prototype.slice.call(arguments, 1);
+  Firestore.log = function(methodName, requestTag, varargs) {
```

Comment:
Should this be specified as `methodName, requestTag, logMessage, varargs` for clarity? It took me a second to figure out that the logMessage is also part of varargs here.

Reply:
Updated.